lib/src/adaptive/adaptive_test.dart

```diff
@@ -70,26 +70,26 @@ extension Adaptive on WidgetTester {
     Key?
         byKey, // Sometimes we want to find the widget by its unique key in the case they are multiple of the same type.
     bool waitForImages = true,
+    String Function()? pathBuilder,
```

Comment:
I did this one month ago, and I really don't remember why I used a function instead of a string. 
But when I see it now, I think it is possible to use a string there.